eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpJsonItemReader.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.accession.dbsnp.io;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.item.ItemReader;
+import uk.ac.ebi.eva.accession.dbsnp.model.DbsnpJson;
+import uk.ac.ebi.eva.accession.dbsnp.parameters.InputParameters;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Item reader for Dbsnp Json import job
+ */
+public class DbsnpJsonItemReader implements ItemReader<DbsnpJson> {
+
+    private static final Logger logger = LoggerFactory.getLogger(DbsnpJsonItemReader.class);
+
+    private InputParameters parameters;
+
+    public DbsnpJsonItemReader(InputParameters parameters) {
+        this.parameters = parameters;
+    }
+
+    @Override
+    public DbsnpJson read() throws Exception {
+        JsonFactory jsonFactory = new JsonFactory();
+        JsonParser jp = jsonFactory.createJsonParser(new File(parameters.getDbsnp()));
```

Comment:
This method seems to be deprecated since version 2.2. Use the other method suggested [here](https://fasterxml.github.io/jackson-core/javadoc/2.8/com/fasterxml/jackson/core/JsonFactory.html#createJsonParser(byte[]))

Reply:
Nice catch! Thanks!